src/sider/krever-reaktivering/krever-reaktivering.less

```diff
@@ -1,63 +1,65 @@
-.krever-reaktivering {
-  margin-top: 1.5rem;
-
-  &__tittel {
-    text-align: center;
-    .blokk();
-  }
-
-  &__infopanel {
-    display: flex;
-    text-align: left;
-    position: relative;
-    justify-content: center;
-    border: 3px solid @navBlaLighten80;
-    border-radius: 8px;
-    padding: 1rem;
-    .blokk-s();
-  }
-
-  &__tekster {
-    width: 90%;
-  }
-
-  &__handinfo-ikon {
-    position: absolute;
-    height: 100%;
-    left: -44px;
-    top: 0;
-    display: flex;
-    align-items: center;
-
-    > img {
-      display: none;
-      width: 80px;
-    }
-  }
+@import '../../../node_modules/nav-frontend-core/less/core';
```

Comment:
![image](https://user-images.githubusercontent.com/462431/48001369-ab04b300-e108-11e8-8422-5218430d808d.png)

Reply:
Knappen er for tett teksten ovenfor